extensions/sql-bindings/src/sql-bindings.d.ts

```diff
@@ -66,7 +86,7 @@ declare module 'sql-bindings' {
 	/**
 	 * Azure Functions binding type
 	 */
-	export const enum BindingType {
+	export enum BindingType {
```

Comment:
You can't have non-const enums in typings files, why did you remove this?

Reply:
I did this to only return the BindingType on the promptSqlBindingMethod, instead of returning the quickPickItem as well. We use the label to send a telemetry event.